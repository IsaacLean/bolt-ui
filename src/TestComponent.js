import React from 'react';

class TestComponent extends React.Component {
  render () {
    const style = {color: this.props.color};
    return <div>
      <h1 style={ style }>
        { this.props.title }
      </h1>
      <small>Lorem Ipsum</small>
    </div>;
  }
}

export default TestComponent;
