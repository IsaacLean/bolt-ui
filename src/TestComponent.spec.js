import React from 'react';
import TestComponent from './TestComponent';
import renderer from 'react-test-renderer';

test('Search snapshot test', () => {
  const component = renderer.create(<TestComponent />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
