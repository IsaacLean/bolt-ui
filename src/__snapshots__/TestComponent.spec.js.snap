// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search snapshot test 1`] = `
<div>
  <h1
    style={
      Object {
        "color": undefined,
      }
    }
  />
  <small>
    Lorem Ipsum
  </small>
</div>
`;
